ChatApp/View/ViewController.swift

```diff
@@ -0,0 +1,128 @@
+//
+//  ViewController.swift
+//  ChatApp
+//
+//  Created by bakhva  on 19.04.23.
+//
+
+import UIKit
+
+final class ViewController: UIViewController {
+    
+    // MARK: Screen Split View
+    private let dividerView: UIView = {
+        let view = UIView()
+        view.backgroundColor = .yellowViewColor
+        view.layer.masksToBounds = true
+        view.translatesAutoresizingMaskIntoConstraints = false
+       
+        return view
+    }()
+        
+    // MARK: Switcher Button
+    private let switcherButton = SwitcherButton()
+    
+    // MARK: First Chat View
+    private let firstChatView = ChatView()
+    
+    // MARK: Second Chat View
+    private let secondChatView = ChatView()
+    
+    // MARK: ViewDidLoad
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        view.backgroundColor = .systemBackground
+        setUpUI()
+        addSubViews()
+        setUpConstraints()
+    }
+    
+    //MARK: Setting Up Constratins
+    private func setUpConstraints() {
+        NSLayoutConstraint.activate([
```

Comment:
separating different components constraints via marks good. would be better if you separate it in the different functions

P.S. dont forget about constants